storage/migrations/0009_oasis_3_emerald_init.up.sql

```diff
@@ -16,10 +19,56 @@ CREATE TABLE IF NOT EXISTS oasis_3.emerald_rounds
   messages_hash    TEXT NOT NULL,
   in_messages_hash TEXT NOT NULL,
 
+  -- TODO: These should also be NOT NULL, but they're populated separately.
+  num_transactions INTEGER,
+  gas_used         BIGINT,
+  size             INTEGER,
+
   -- Arbitrary additional data.
   extra_data JSON
 );
 
+CREATE TABLE IF NOT EXISTS oasis_3.emerald_transactions
+(
+  round BIGINT NOT NULL,
```

Comment:
cant find where i saw it but there was some comment on how these files were indented. the answer is that it was done manually so feel free to do the same or also feel free to wait until we add a SQL formatter 🤷🏽 

brief search yields that [squawk](https://github.com/sbdchd/squawk) looks ok and has a [github action](https://github.com/sbdchd/squawk-action) to pair in CI

Reply:
Moved to https://github.com/oasisprotocol/oasis-indexer/issues/150

Reply:
proceeding with manual indentation :robot: